### ISSUE TITLE
Display Pool details

### ISSUE THREAD

**Issue by jahabeebs** [open-dollar/od-app#240]
depends on #239 
Part of TVL project https://github.com/open-dollar/od-app/issues/234

Only the UI portion for this issue. The data will be in another issue. Can use hardcoded values for now (eg from a hardcoded object defined at the top of the file)

## Description
- Display details about the pool

![image](https://github.com/open-dollar/od-app/assets/35622595/b7e9fc66-59a0-4cba-b9dc-e14a896bf89e)
